config/projects.yml

```diff
@@ -259,7 +259,7 @@ media-bundle:
       php: ['5.3', '5.4', '5.5', '5.6', '7.0']
       target_php: 5.6
       versions:
-        symfony: ['2.3', '2.7', '2.8', '3.0', '3.1']
+        symfony: ['2.8', '3.0', '3.1']
```

Comment:
@jordisala1991 : the mistake happened here. Can you do a PR on the admin bundle, just to see if the build would be green if we restored those?